ocaml/xapi/test_vdi_cbt.ml

```diff
@@ -282,12 +283,32 @@ let test_vdi_after_data_destroy () =
 (* check VDI.data_destroy throws VDI_NOT_MANAGED if managed:false *)
 let test_vdi_managed_data_destroy () =
   let __context = Test_common.make_test_database () in
-  let _,vDI = make_mock_server_infrastructure ~__context in
+  let sR,vDI = make_mock_server_infrastructure ~__context in
+  register_smapiv2_server ~vdi_data_destroy:(fun _ ~dbg ~sr ~vdi -> ()) (Db.SR.get_uuid ~__context ~self:sR);
   Db.VDI.set_managed ~__context ~self:vDI ~value:false;
   OUnit.assert_raises ~msg:"VDI.data_destroy only works on managed VDI"
     Api_errors.(Server_error (vdi_not_managed, [Ref.string_of vDI]))
     (fun () -> Xapi_vdi.data_destroy ~__context ~self:vDI)
 
+(* behaviour verification VDI.list_changed_blocks *)
+let test_vdi_list_changed_blocks () =
+  let __context = Test_common.make_test_database () in
+  let list_changed_blocks_params = ref None in
+  let sR,vdi_from = make_mock_server_infrastructure ~__context in
+  let vdi_to = Test_common.make_vdi ~__context ~sR () in
+  list_changed_blocks_params := Some (sR,vdi_from,vdi_to);
+  Db.VDI.set_cbt_enabled ~__context ~self:vdi_to ~value:true;
```

Comment:
I think we should set `cbt_enabled` to true on `vdi_from` as well.

Reply:
Already done by the `make_mock_server_infrastructure` function which creates the VDI.